Formula/mapmap.rb

```diff
@@ -0,0 +1,34 @@
+class Mapmap < Formula
+  desc "Video mapping software"
+  homepage "https://mapmapteam.github.io/"
+  url "https://github.com/mapmapteam/mapmap/archive/0.6.2.tar.gz"
+  sha256 "2f1b8236d448b6839395fd78b1d407c034749a549666270f2a17e6b5f2d25c78"
+  head "https://github.com/mapmapteam/mapmap.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "gst-plugins-base"
+  depends_on "gstreamer"
+  depends_on "liblo"
+  depends_on "qt"
+  depends_on "gst-libav" => :recommended
+  depends_on "gst-plugins-good" => :recommended
+
+  def install
+    inreplace "src/src.pri" do |s|
```

Comment:
These string replacements should not be required for a successful build. If they are, that needs to be fixed upstream.